pkg/encrypt/server-side.go

```diff
@@ -144,3 +157,18 @@ type s3 struct{}
 func (s s3) Type() Type { return S3 }
 
 func (s s3) Marshal(h http.Header) { h.Set(sseGenericHeader, "AES256") }
+
+type kms struct {
+	key     string
+	context *string
```

Comment:
Can we use a bool var - like `contextPresent` instead of a `*string` and nil checks?